cmd/config/set.go

```diff
@@ -0,0 +1,30 @@
+package config
+
+import (
+	"github.com/chelnak/pdk/internal/config"
+	"github.com/spf13/cobra"
+)
+
+var (
+	key   string
+	value string
+)
+
+func getSetCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set",
+		Short: "Sets a configuration property to the specified value value.",
+		Long:  "Sets a configuration property to the specified value value.",
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
This could be refactored into a `run` function

Reply:
Actually I noticed an issue with our assumption.

This works until you start to define multiple commands in the same package.

Because `func run()` is package scoped it can only appear once!

Part of me felt it might be messy having lots of methods called runSomething for each command..

I guess the alternative would be to create a package for each subcommand in this instance but is that overkill?